screens/Maps/EventMap.tsx

```diff
@@ -1,10 +1,11 @@
 import MapView, { Callout, Marker, PROVIDER_GOOGLE } from "react-native-maps"
-import React from "react"
+
 import { RouteProp, useNavigation, useRoute } from "@react-navigation/native"
 import { View, Text, TouchableOpacity } from "react-native"
 import styles from "./styles" // Import styles
 import { Ionicons } from "@expo/vector-icons"
 import { Event } from "../../types/Event"
+import React = require("react")
```

Comment:
Why do you import react this way? does it differ from the usual import? It always break my code so if it does work for you I recommend you changing it before pushing

Reply:
This should not even be allowed by the linter...